test/TestTaskIncludes.py

```diff
@@ -1,5 +1,4 @@
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.